app/scripts/lib/stores/base-store.ts

```diff
@@ -41,8 +53,8 @@ export type MetaMaskStorageStructure = {
  * storage system. This method should handle necessary validation or
  * error handling to ensure the state is persisted correctly.
  */
-export abstract class BaseStore {
-  abstract set(state: MetaMaskStateType): Promise<void>;
+export type BaseStore = {
```

Comment:
I don't think this needs the extra overhead of an `abstract class`. I think its fine to change it. Thoughts? Objections?

Reply:
Good call